p3387.tex

```diff
@@ -75,6 +75,10 @@
 \section{Introduction}
 \label{intro}
 
+???
+%IDS - I note that you capitalise "Contracts" but not "Coroutines" in the paper - it's not important but does catch the eye...
```

Comment:
In SG21, we at some point started capitalising Contracts to distinguish the proposed language facility (spelled with a capital C) from the general software design concept of a "contract" (spelled with a lowercase C). No such convention exists for coroutines, which is why I didn't capitalise "coroutines". Any other ideas how this could be resolved?

Reply:
it's not important - just caught my eye while reading.